docs/development.md

```diff
@@ -22,4 +22,12 @@ slug: /development
     heading="SDKs"
     href="/sdks"
   />
+  <card
+    heading="Postman Collection"
+    href="https://www.postman.com/macrometa"
+  />
+  <card
+    heading="Jupyter Notebooks"
```

Comment:
Have you tested these workbooks lately? Because when I last tried them, they were not working well. I recommend not drawing attention to them until they have been thoroughly reviewed.